packages/network/src/index.ts

```diff
@@ -73,15 +67,9 @@ export class Network extends Connector {
   public async activate(desiredChainId = this.defaultChainId): Promise<void> {
     if (!this.provider) this.actions.startActivation()
 
-    this.provider = await this.isomorphicInitialize(desiredChainId)
+    this.customProvider = await this.isomorphicInitialize(desiredChainId)
```

Comment:
Can this promise reject? If so, `.catch` and `this.actions.reportError`.

Reply:
neither this nor the other instance should be able to throw, but on the off chance that ethers errors in the constructor of JsonRpcProvider or something, i'll add a catch!